datalad/metadata/tests/test_base.py

```diff
@@ -78,15 +78,20 @@
 
 @with_tempfile(mkdir=True)
 def test_get_metadata_type(path):
-    Dataset(path).create()
+    ds = Dataset(path).create()
     # nothing set, nothing found
-    assert_equal(get_metadata_type(Dataset(path)), [])
+    assert_equal(get_metadata_type(ds), [])
     # got section, but no setting
     open(opj(path, '.datalad', 'config'), 'w').write('[datalad "metadata"]\n')
```

Comment:
Just a side note, not related to the fix in this PR: It seems add to me that we use `open(..., 'w')` here.  That clobbers the dataset ID config.

Reply:
indeed!